pages/Documents/MessagingChannels/MobileAppMessagingSDKforAndroid/ReleaseNotes/5.13.0.md

```diff
@@ -0,0 +1,41 @@
+**Release date:** November 10, 2022
+
+# Overview
+
+Android Mobile Messaging SDK version 5.13.0 release includes new features, enhancements and bug fixes.
+
+## Environment requirements
+
+The Android Mobile Messaging SDK version 5.13.0 uses:
+
+- Minimum API version 21
+- Compile API version 33
+- Target API version 33
+- Maps SDK "com.google.android.gms:play-services-maps:17.0.1"
+- Structured Content Library “com.liveperson.android:lp_structured_content:2.4.0”
+- Date Picker Library “com.liveperson.android:lp-date-picker:2.0.1”
+- Schedule Slot List Library "com.liveperson.android:lp-appointment-scheduler:2.0.0"
+
+## New features
+
+### Capability to turn on/off voice sharing at a run-time
```

Comment:
Recommended but optional (in this PR): Please keep a blank line between headings and succeeding content.